lib/_stream_readable.js

```diff
@@ -120,6 +120,9 @@ function ReadableState(options, stream, isDuplex) {
   // True if the error was already emitted and should not be thrown again
   this.errorEmitted = false;
 
+  // True if close has been emitted or should have been given emitClose.
+  this.closed = false;
```

Comment:
Is this property public, should it be documented?

Otherwise, considering using a symbol:
```js
const kClosed - Symbol('kClosed')

// ...

this[kClosed] = // ...
```

Reply:
> Is this property public, should it be documented?

I don't think its any more or less public than all the other properties on the stream state?

Reply:
I kind of get your suggestion but it feels a bit inconsistent.